CHANGELOG.md

```diff
@@ -45,6 +45,9 @@ Current
 
 ### Changed:
 
+- [Fix wrong default druid url and broken getInnerMostQuery](https://github.com/yahoo/fili/pull/528)
+    * Comment out the wrong default druid broker url in module config, add check for validate url in `DruidClientConfigHelper `
```

Comment:
Add a few more words to explain that having the default specified was causing the app to fail to do backwards compatibility.